CONTRIBUTING.md

```diff
@@ -49,6 +49,23 @@ close your PR in order to prevent work being done on it.
 [issues]: https://github.com/rust-windowing/winit/issues
 [matrix]: https://matrix.to/#/#rust-windowing:matrix.org
 
+### Formatting
+
+Winit strictly relies on nightly `rustfmt`. A PR will not even get reviewed if
+nightly `cargo-fmt` check is not passed. If you have `rustup` installed you can
+install nightly toolchain using the following command `rustup toolchain install
+nightly`.
+
+Then to format the code you can run `cargo +nightly fmt`.
+
+For more on how to manage toolchains, see [the textbook](https://rust-lang.github.io/rustup/concepts/toolchains.html).
+
+However, if you feel uncomfortable using `nightly` toolchain, ensure that CI
+passes with the exception of formatting and explicitly indicate that your 
+code should be formatted before merge.
```

Comment:
Nit:
- "However" sounds weird in this context
- "explicitly indicate" may be confusing (where do I explicitly indicate such a thing?), tell people to just ask on the PR

```suggestion
If you feel uncomfortable using a `nightly` toolchain, ensure that CI passes
with the exception of formatting, and ask the maintainers on your PR to format
the code before merge.
```